.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+defaults: &defaults
+    working_directory: ~/repo
+
+jobs:
+  build:
+    # Run tests inside a docker container set up on CircleCI
+    docker:
+      - image: circleci/node:11.6.0
+
+    # Steps to run tests
+    steps:
+      # Checkout the code
+      - checkout
+
+      - run:
+          name: test-reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+
+      # Update npm
+      - run:
+          name: update-npm
+          command: sudo npm i -g npm@latest
+
+      # Use cached node_modules installations
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-pee-wee
+          command: npm install
+
+      # Save node_modules to cache to speed up subsequent builds
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+
+      # Run tests
+      - run:
+          name: tests
```

Comment:
There is a pending command for CircleCI to report the coverage back to code climate. 
You can refer to this doc
https://docs.codeclimate.com/docs/circle-ci-test-coverage-example